src/libasr/asr_utils.h

```diff
@@ -1170,6 +1170,39 @@ static inline bool extract_value(ASR::expr_t* value_expr,
     return true;
 }
 
+static inline bool extract_value(ASR::expr_t* value_expr,
```

Comment:
Is this only extracting string values? If so, let's name it:
```suggestion
static inline bool extract_string_value(ASR::expr_t* value_expr,
```

Reply:
Yes, updated.